app/views/admin/shared/_filter_options.html.erb

```diff
@@ -43,6 +43,20 @@
       } %>
     <% end %>
 
+    <% if filter_by.include?(:organisation_id) %>
```

Comment:
What's the difference between this one and the filter_by organisation below?

Reply:
I had copied it over from the original `StatisticsAnnouncement` filter options. Changing it to `organisation` doesn't seem to impact functionality, not sure why it was referred to as `organisation_id` before in the controller and the filter model.